examples/java/src/main/java/org/apache/beam/examples/complete/kafkatopubsub/options/KafkaToPubsubOptions.java

```diff
@@ -90,6 +91,7 @@
 
   @Description(
       "Additional kafka consumer configs to be applied to Kafka Consumer (e.g. key1=value1;key2=value2).")
+  @Nullable
```

Comment:
@kennknowles do you think there's anything we can do to enforce that pipeline options without a default are `Nullable`? If we'd had that checkerframework would have caught this.

Reply:
yes, I think this would be pretty easy in `PipelineOptionsFactory`. The code there is not particularly well factored but this is probably a surgical fix.

Reply:
Would we just add the annotation in some generated code there? I took a quick look and couldn't find a way to do it. I filed BEAM-13758 to track this and linked it to BEAM-10402.